stackslib/src/clarity_vm/database/mod.rs

```diff
@@ -66,6 +71,21 @@ impl GetTenureStartId for StacksDBConn<'_> {
             .map(|block_id| TenureBlockId::from(block_id)))
     }
 
+    fn get_tenure_block_id_at_cb_height(
```

Comment:
Minor nit: can you spell out `coinbase` instead of `cb`?

Reply:
I'd prefer to add a rustdoc to this trait method rather than lengthening an already pretty long function name.

Reply:
Sure, please do

Reply:
Added to the trait definition (https://github.com/stacks-network/stacks-core/pull/5321/commits/d901503496a4c91671f9700a0e4df972beb1aafd#diff-f8f75e1cce41646d74991a58ef25998c39a02eaebb3792d37ac78dcc13d8e17fR50)